packages/client/src/views/PrintCertificate/ReviewCertificateAction.tsx

```diff
@@ -360,7 +360,12 @@ function mapStatetoProps(
         ...draft.data,
         template: {
           ...draft.data.template,
-          ...(isCertificateForPrintInAdvance(draft) && { printInAdvance: true })
+          ...(isCertificateForPrintInAdvance(draft) && {
+            printInAdvance: true
+          }),
+          ...(draft.data.deceased?.ageOfIndividualInYears && {
+            ageOfDeceasedInYears: draft.data.deceased.ageOfIndividualInYears
+          })
```

Comment:
@Nil20 Could we use a template mapper function instead of this?

Reply:
Yeah, done!

Reply:
Closing this PR, then.